scio-test/src/test/scala/com/spotify/scio/testing/TestingUtils.scala

```diff
@@ -23,6 +23,7 @@ import scala.collection.JavaConverters._
 
 private[scio] object TestingUtils {
   // Value type Iterable[T] is wrapped from Java and fails equality check
+  @deprecated // Kept for backward compatibility as a test case only
```

Comment:
This is in `private[scio]` so we can safely remove. Those who hack to use it are on their own to refactor.

Reply:
good point ... I didn't realized that it was private[scio]